tests/phpunit/includes/testcase.php

```diff
@@ -41,4 +41,27 @@ protected function assert_has_language( $object, $language, $message = '' ) {
 		$this->assertNotFalse( $object_language, $message );
 		$this->assertSame( $language, $object_language->slug, $message );
 	}
+
+	/**
+	 * Verifies that a file exists.
+	 * Depending on the environment variable `GITHUB_ACTIONS`:
+	 * - If defined (value in GitHub Actions is `'true'`): does an assertion.
+	 * - If not defined: skips if the file doesn't exist.
+	 *
+	 * @see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
+	 *
+	 * @param string $path    Path to the file.
+	 * @param string $message Error message.
+	 * @return void
+	 */
+	protected static function markTestSkippedIfFileNotExists( string $path, string $message = '' ): void {
```

Comment:
Done in [ebe6590](https://github.com/polylang/polylang/pull/1581/commits/ebe65903da128eef1a5182c090d1ddb4c82c1f46), so it can be used in AJAX testcases and others.